contribs/gnogenesis/internal/txs/txs_add_packages.go

```diff
@@ -53,10 +70,12 @@ func execTxsAddPackages(
 		return errInvalidPackageDir
 	}
 
+	creator := crypto.MustAddressFromString(cfg.deployerAddress)
```

Comment:
What do you think about making this a package global?
It's a small command package, so I see no harm in doing so

Reply:
yeah I do not see a problem with that :)
changed on c80eb7c622532c7b6ab1dd1be0cd79b317182ab2